lib/components/slider/column-config-slider.jsx

```diff
@@ -149,41 +172,117 @@ export const ColumnConfigSlider = (props) => {
+											disabled && marker >= value[0] - 1 && marker < value[1] && 'es:bg-secondary-300',
+											disabled && !(marker >= value[0] - 1 && marker < value[1]) && 'es:bg-secondary-400',
+											showOuterAsGutter && (i === 0 || i === columns - 1) && 'es:hidden',
 										)}
-										style={{
-											gridColumn: `${state.values[i]} / span 1`,
-										}}
+										style={{ gridColumn: i + 1 }}
 									/>
 								))}
+
+								{state.values.map((_, i) => {
+									let gridCol = `${state.values[i] - (i === 0 ? 1 : 0)} / span 2`;
+
+									if (i === 0 && state.values[0] === 1) {
+										gridCol = '1';
+									}
+
+									if (i === 1 && state.values[1] === columns) {
+										gridCol = `${columns}`;
+									}
+
+									return (
+										<ReactAriaSliderThumb
+											key={i}
+											index={i}
+											aria-label={thumbLabels?.[i]}
+											isDisabled={(i === 0 && disableOffset) || (i === 1 && disableWidth)}
+											className={clsx(
+												'es:static! es:row-span-1 es:row-start-1 es:h-full es:w-1 es:rounded-md es:transition es:duration-300',
+												'es:origin-center',
+												'es:hover:ring-[0.25px] es:focus-visible:ring-[0.5px] es:dragging:ring-[1px] es:ring-accent-500',
+												'es:transform-none!',
+												'es:dragging:es:bg-accent-600 es:disabled:bg-secondary-400',
+												'es:focus-visible:outline-2 es:outline-offset-2 es:outline-accent-500/40',
+												'es:bg-accent-500',
+												'es:hover:ring-accent-600',
+												'es:dragging:bg-accent-600 es:dragging:ring-accent-600',
+												!disabled && 'es:hover:not-dragging:cursor-grab',
+												i === 0 && disableOffset && 'es:hidden',
+												i === 1 && disableWidth && 'es:hidden',
+												'es:justify-self-center',
+												!flat && !disabled && 'es:shadow-xs es:shadow-black/5',
+											)}
+											style={{
+												gridColumn: gridCol,
+											}}
+										>
+											<AnimatePresence>
+												{i === state.focusedThumb && (
+													<motion.div
+														className={clsx(
+															'es:absolute es:bottom-12 es:w-fit es:min-w-5 es:h-6 es:text-nowrap es:-translate-x-1/2',
+															'es:bg-surface-50/80 es:text-surface-700',
+															'es:backdrop-blur-xs',
+															'es:text-12 es:leading-none',
+															'es:py-1 es:px-2 es:rounded-lg',
+															'es:text-center',
+															'es:line-clamp-1',
+															'es:flex es:items-center es:justify-center es:gap-1',
+															'es:shadow',
+															showOuterAsGutter && 'es:icon:size-2 es:icon:stroke-3',
+														)}
+														initial={{ y: 2, opacity: 0, scale: 0.85 }}
+														animate={{ y: 0, opacity: 1, scale: 1 }}
+														exit={{ y: 2, opacity: 0, scale: 0.85 }}
+													>
+														{i === 0 && showOuterAsGutter && state.values[0] > 1 && sprintf(__('From %d', 'infobip2024'), state.values[i] - 1)}
```

Comment:
```suggestion
														{i === 0 && showOuterAsGutter && state.values[0] > 1 && sprintf(__('From %d', 'eightshift-ui-components'), state.values[i] - 1)}
```
Check for other instances of this one 😉